exam.cls

```diff
@@ -120,9 +100,13 @@
 {%
   \setlength{\tabcolsep}{0.2em}%
   \begin{tabular}{rl}
-    \textbf{\small NOM :} & \zsavepos{lastnamebox}\multido{}{\nbBoxesHeaderLine}{~\boxStudentID}\\
-    \textbf{\small PRÉNOM :} & \zsavepos{firstnamebox}\multido{}{\nbBoxesHeaderLine}{~\boxStudentID}\\
-    \ifheaderWithID \textbf{\small NUM ETD :} & \zsavepos{stdidbox}\multido{}{\nbBoxesStudentID}{~\boxStudentID} \else \\  \fi
+    \ifanonymousExamen%
+      \textbf{\small \No ANONYMAT~:} & \zsavepos{stdidbox}\multido{}{\nbBoxesAnonimityNumber}{~\boxStudentID}\\
+    \else
+      \textbf{\small NOM :} & \zsavepos{lastnamebox}\multido{}{\nbBoxesHeaderLine}{~\boxStudentID}\\
```

Comment:
Does it work with the english template?

Reply:
But may be part of another PR: add another parameter to define the language of the template

Reply:
Is there an English template?